### ISSUE TITLE
"Show full special rule text" option in the table view remove the rules text at the bottom 

### ISSUE THREAD

**Issue by BaptisteD** [AdamLay/opr-army-forge#387]
In the table view, enable the option remove the Special Rules card (like in the Card view). But in this case, you don't have the rules' text anywhere then. 
Should we change the behavior of this option in this case ?

![Screenshot_20230106-212544](https://user-images.githubusercontent.com/2566847/211093917-d0761e41-2086-428f-a696-16e44e65c6a1.jpg)

![Screenshot_20230106-212552](https://user-images.githubusercontent.com/2566847/211093936-4dffc82a-443a-4bcf-8498-ec3bf656f76e.jpg)

**Comment by AdamLay** [comment]
Yes good spot - it seems this is really an option only relevant to the card view!
